Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -811,6 +865,47 @@ def check_font_has_a_license(fb, licenses):
 
 @register_test
 @old_style_test(
+    id='com.google.fonts/test/029'
+  , conditions=['license']
+  , priority=CRITICAL
+)
+def check_copyright_entries_match_license(fb, ttFont, license):
+  """Check copyright namerecords match license file"""
+  failed = False
+  for license_filename in ['OFL.txt', 'LICENSE.txt']:
```

Comment:
I think we can get rid of the complete for loop, because in the `licenses` condition we already check that the licenses are in `['OFL.txt', 'LICENSE.txt']` and in the `license` condition we make sure that there's only a single license.

https://github.com/felipesanches/fontbakery/blob/047532d048d6cbc55c835cdfa8809ebc9b07b346/Lib/fontbakery/specifications/googlefonts.py#L832

Also, in general. the for loop is a poor choice here, since a:

```
if license in ['OFL.txt', 'LICENSE.txt']:
  placeholder = PLACEHOLDER_LICENSING_TEXT[license]
```

Would do the same job but more direct.

Much better though would be doing directly:

```
if license in PLACEHOLDER_LICENSING_TEXT:
  placeholder = PLACEHOLDER_LICENSING_TEXT[license]
```

and even better than that would be doing directly:

```
placeholder = PLACEHOLDER_LICENSING_TEXT[license]
```
The latter will raise a KeyError if `license` is not in `PLACEHOLDER_LICENSING_TEXT` and that will result in an `ERROR` status, which is good, because then we know that we identified something as a license (in the license condition, but we don't have a placeholder thing for it. It means our spec must be improved.